test/snippet/alphabet/quality/phred94.cpp

```diff
@@ -0,0 +1,15 @@
+#include <seqan3/alphabet/quality/phred94.hpp>
+#include <seqan3/core/debug_stream.hpp>
+
+int main()
+{
+    seqan3::phred94 phred;
+    phred.assign_rank(2); // wrapper for assign_phred(2)
+    seqan3::debug_stream << static_cast<int>(phred.to_phred()) << "\n"; // 2
+    seqan3::debug_stream << phred.to_char() << "\n";        // '#'
+    seqan3::debug_stream << static_cast<int>(phred.to_rank()) << "\n";  // 2
```

Comment:
Is to_rank and to_phred the same?

Reply:
In this case, yes. But not for every phred score alphabet, see ```seqan3::phred68legacy```. When you have a look in the Table of the all.hpp you will recognize that the score could be negative.